docroot/themes/custom/mass_admin_theme/mass_admin_theme.theme

```diff
@@ -227,10 +227,22 @@ function _add_description_display_before(&$variables) {
  */
 function mass_admin_theme_preprocess_form_element(&$variables) {
   _add_description_display_before($variables);
+
+  // Remove cookie when in the admin theme to ensure that the cookie is reset.
+  setrawcookie('ah_app_server', '', REQUEST_TIME - 3600, '/');
```

Comment:
A more API-ish way to do this is https://drupal.stackexchange.com/a/293281/549 but not a big deal at all.

Reply:
Thanks. Adjusted.